compiler/src/dmd/tokens.d

```diff
@@ -268,6 +268,8 @@ enum TOK : ubyte
     _Noreturn,
     _Static_assert,
     _Thread_local,
+	wchar_t,
+    char32_t,
```

Comment:
need to add these to `extern (D) private static immutable string[TOK.max + 1] tochars =`